bin/docker_start.sh

```diff
@@ -1,37 +1,35 @@
 #!/bin/sh
 
-set -e  # exit on error
+set -ex
 
 # Wait for the database container
 # See: https://docs.docker.com/compose/startup-order/
-db_host=${DATABASE_HOST:-db}
-db_user=${DATABASE_USER:-postgres}
-db_password=${DATABASE_PASSWORD}
+db_host=${DB_HOST:-db}
+db_user=${DB_USER:-postgres}
+db_password=${DB_PASSWORD}
+db_port=${DB_PORT:-5432}
 
-until PGPASSWORD=$db_password psql -h "$db_host" -U "$db_user" -c '\q'; do
+uwsgi_port=${UWSGI_PORT:-8000}
+
+until PGPORT=$db_port PGPASSWORD=$db_password psql -h "$db_host" -U "$db_user" -c '\q'; do
   >&2 echo "Waiting for database connection..."
   sleep 1
 done
 
 >&2 echo "Database is up."
 
-# Collect static files
```

Comment:
Er staat me iets van bij dat dit expres NA het opkomen van de database moest gebeuren... Weet alleen de case niet meer. Prima om het in de Docker te doen. Wordt wel een "oh ja" momentje later.

Reply:
Is ook zo gedaan in default-project, en build prima. Er worden alleen even tijdelijk een settings module & secret key gezet, misschien was dat het?